app/models/story_parser.rb

```diff
@@ -989,8 +990,8 @@ def convert_revised_at(date_string)
           # probably seconds since the epoch
           date = Time.at($1.to_i)
         end
-        date ||= Date.parse(date_string)
-        return '' if date > Date.today
+        date ||= DateTime.parse(date_string)
```

Comment:
Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.